Source/Core/FeatureDetection.js

```diff
@@ -308,6 +308,12 @@ var FeatureDetection = {
   typedArrayTypes: typedArrayTypes,
 };
 
+function supportsBasis(scene) {
+  return FeatureDetection.supportsWebAssembly() && scene.context.supportsBasis;
+}
+
+FeatureDetection.supportsBasis = supportsBasis;
```

Comment:
What's the right way to add docs here? It thinks `supportsBasis` is a member of `FeatureDetection` rather than a method. Maybe something like this?
```js
FeatureDetection.supportsBasis = function (scene) {
    return supportsBasis(scene);
}
```

Reply:
Actually, why can't it be:

```js
FeatureDetection.supportsBasis = function (scene) {
  return FeatureDetection.supportsWebAssembly() && scene.context.supportsBasis;
}
```